src/zui/ZUIAvatar/index.tsx

```diff
@@ -14,12 +15,19 @@ const SIZES = {
 };
 
 const ZUIAvatar = forwardRef<HTMLDivElement, ZUIAvatarProps>(
-  ({ orgId, personId, size = 'md', ...restProps }, ref) => {
+  (
+    { orgId, personId, isUserAvatar = false, size = 'md', ...restProps },
+    ref
+  ) => {
     return (
       <Avatar
         ref={ref}
         {...restProps}
-        src={`/api/orgs/${orgId}/people/${personId}/avatar`}
+        src={
+          isUserAvatar
+            ? `/api/users/${personId}/avatar`
```

Comment:
Note how `orgId` is not used here, and `personId` is used to indicate the _user ID_ even though it's called `personId`. That's telling me that this component is not a good fit for the use case of showing user avatars.